client/src/app/tabs/getEditMenu.js

```diff
@@ -367,12 +367,10 @@ export function getUndoRedoEntries({
 }) {
   return [ {
     label: 'Undo',
-    accelerator: 'CommandOrControl+Z',

```

Comment:
Could you elaborate why we do this? This removes the shortcut from the menu.

Reply:
As a result it breaks undo/redo, at least for forms.

Reply:
Accelerators should be brought back, but there is a bug we need to fix.

Without the accelerators, undo/redo works fine for me both from keyboard and menu.

With the accelerators, undo/redo trigger twice every time it's run from keyboard shortcut.

Here is the recording of how it works with accelerators untouched when undoing from keyboard:

https://github.com/user-attachments/assets/2352e3fb-474d-4bbd-9623-dfb9941606a3

Reply:
Why do we bind the keyboard natively? I thought this would be handled via the menu and we'd simply is `canvas.isFocused()` to control activation?

Reply:
Done via https://github.com/camunda/camunda-modeler/pull/4620/commits/7e1ab79fea26eb6e86dbebc85f731463d91181d0 and https://github.com/camunda/camunda-modeler/pull/4620/commits/bccc93ec29be03a8a7a586e53f5c9606796105c9.

Reply:
We'd rather keep the accelerator but disable the registration so that the shortcut is still displayed but handled in the browser window: https://www.electronjs.org/docs/latest/api/menu-item#menuitemregisteraccelerator

Reply:
@barmac I think we don't even need the special property, cf. https://github.com/camunda/camunda-modeler/pull/4620#discussion_r1847743825.

Reply:
OK so you will never add event listeners, but rather use the electron menu to trigger the action? I am OK with that.

Reply:
YES.